static/src/js/components/SearchPanels/SearchPanels.js

```diff
@@ -177,10 +177,10 @@ class SearchPanels extends PureComponent {
 
     const {
       pageNum: collectionsPageNum = 1,
-      sortKey: collectionsSortKey = ''
+      sortKey: collectionsSortKey = '-score'
```

Comment:
All of the places where we have a string value for a sort key would benefit from creating a constant. So instead of typing/copying `-score` everywhere, you could do something like `CollectionSortKeys.ScoreDescending`. This makes writing the code a little easier since you can get autocomplete from the editor to help you with the types of values to expect